processors/nitf21GeospatialConverters/decimalDegreeConverter.ts

```diff
@@ -84,7 +71,7 @@ export class DecimalDegreeConverter extends BaseProcessor {
                 }
 
                 if (arrCoords.length > 0) {
-                    const nitf21Helper = new Nitf21ConverterHelper(this.executionContext, this.processorDef);
+                    const nitf21Helper: Nitf21ConverterHelper = new Nitf21ConverterHelper(this.executionContext, this.processorDef, this.logger);
```

Comment:
It's not TypeScript without typing... thanks for the catch.

Reply:
Actually, this was an oversite on my part =P The Type in this case is inferred by the constructor e.g. not necessary. I added it while doing the migration work and forgot to take the type out. I had it only to help me quickly "jump to reference". I will remove it since it is unnecessary.

Reply:
Removed the strong typing as it was redundant.